components/Players/PlayerManager.vue

```diff
@@ -287,6 +289,8 @@ export default Vue.extend({
             this.setLastProgressUpdate({ progress: 0 });
 
             this.resetMetadata();
+
+            this.removeMediaHandlers();
```

Comment:
Okay, it works now, but this makes the controls blink:


https://user-images.githubusercontent.com/10274099/106885830-7a46c600-66e3-11eb-9878-ac83f5a948fd.mp4

I would create a ``updateMetadata`` method that updates the metadata instead of plain removing the handlers and clearing the metadata. I don't see why we need a ``resetMetadata`` one, that ``updateMetadata`` method should clear the metadata that's missing of the item, but if we want to clear up everything, it's because the playing session ended, so we're better calling ``removeMediaHandlers`` and that's it.

That ``updateMetadata`` method can also take handle of removing the actions of the previous/next track if there's no next/previous track in the queue.

Reply:
This is only called when playback is stopped, not we navigate between tracks.

I can't seem to reproduce this. Does this still occur in chrome

Reply:
@Camc314 True, GH code preview confusing me a bit with the collapses. See how Spotify app works:

https://user-images.githubusercontent.com/10274099/106888164-7799a000-66e6-11eb-83c3-0dc613dabe37.mp4

I added console.log calls to ``removeMediaHandler`` and ``updateMetadata`` and I can confirm that ``updateMetadata`` is called only once per track change and ``removeMediaHandler`` is also called only when stopping.

Tested Chrome 88 in browserstacks and I can't reproduce the issue, so this does seem indeed and Edge Chromium bug.